tests/test_rtd_github_links.py

```diff
@@ -149,6 +150,11 @@ def test_as_function(
     assert github_url(f"scanpydoc.{module}.{name}") == f"{prefix}/{obj_path}#L{s}-L{e}"
 
 
+def test_no_line_nos_for_builtin_source() -> None:
```

Comment:
Open to other names here, surely there is a better word for things like `collections.abc` vs `dict` (which would raise a `TypeError`)